setup.py

```diff
@@ -2,7 +2,23 @@
 from setuptools import setup
 from setuptools import Command
 
-__version__ = version = VERSION = '0.1'
+
+def _get_version():
+    """Returns the PROTEUS software version
```

Comment:
```suggestion
    """Returns the PROTEUS software version from the file `src/proteus/version.py`
```

Reply:
@gshiroma Wanted to check that you saw this? Or was there a concern? I can understand how there's a discrepancy in using a UNIX-style file path in the docstring, but then generalizing with os.path.join() in the code itself, so perhaps there's a 3rd solution that is better. Thanks!

Reply:
I missed this comment. I think I prefer the original, but I don't mind adding the extra text.